Jenkinsfile

```diff
@@ -58,7 +58,7 @@ pipeline {
                     }
                     steps {
                         withDockerRegistry(credentialsId: "${DOCKER_CREDENTIALS_ID}", url: "") {
-                            sh 'sh build-n-publish.sh --image=${DOCKER_IMAGE_NAME} --commit=${GIT_COMMIT} --name=pr-${CHANGE_ID}'
+                            sh 'sh build-n-publish.sh --image=${DOCKER_IMAGE_NAME}${PACKAGE_SUFFIX} --commit=${GIT_COMMIT} --name=pr-${CHANGE_ID}'
```

Comment:
Where is defined the value of `PACKAGE_SUFFIX` ?

What do you think about append the suffix when set the value of `DOCKER_IMAGE_NAME` ? That will allow change only one line

Reply:
In my mind, it is explicit and clear, and I already did it in this way in a lot of repositories, so, I prefer to keep it as it is here. But you can do it in that way in your repositories.

Take into account that it is a temporal patch.

Reply:
I applied your suggestion in [MSEditor](https://github.com/MakingSense/MSEditor/pull/1489/files#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8R25) and [Forms](https://github.com/MakingSense/doppler-forms/pull/424/files#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8R51).

You are right, it looks better.